.github/workflows/ci.yml

```diff
@@ -25,9 +25,11 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -r requirements.txt
-          pip install pylint
+          pip install -r requirements-dev.txt
       - name: Run Pylint
         run: ./lint --disable=W
+      - name: Check formatting
+        run: black --check gnomad
```

Comment:
Do you think we need `--target-version=py36` here? It didn't appear to impact any of the code in the repo but just curious.

Reply:
It's specified in [pyproject.toml](https://github.com/broadinstitute/gnomad_methods/blob/master/pyproject.toml).

https://black.readthedocs.io/en/stable/pyproject_toml.html